Form/Type/BasePickerType.php

```diff
@@ -125,6 +125,7 @@ protected function getCommonDefaults()
             'widget' => 'single_text',
             'datepicker_use_button' => true,
             'dp_pick_time' => true,
+            'dp_pick_date' => true,
```

Comment:
@jlamur not sure there is a need for the expression you proposed if there is this default value.

Reply:
pickDate = true, this is standard behavior sonata_type_datetime_picker. I just expanded the option to turn off date selection.

Reply:
I though about this hypothetical guy who extends this class, overrides the `getCommonDefaults()` method and doesn't call `parent::getCommonDefaults()` :thinking:

We will let this guy deal with this on its own. :)

Reply:
Good thinking, but that's a bit like spacebar heating :P